watcher.go

```diff
@@ -325,15 +326,18 @@ func (w *Watcher) Watch(ctx context.Context, tmplCh chan string) error {
 	}
 }
 
-// Buffer sets the template to activate buffer and accumulate changes for a
+// Buffering sets the template to activate buffer and accumulate changes for a
 // period. If the template has not been initalized or a buffer period is not
-// configured for the template, it will skip buffering.
-func (w *Watcher) Buffer(n Notifier) bool {
-	// first pass skips buffering.
-	if !w.tracker.notifierTracked(n) {
-		return false
+// configured for the template, it will skip the buffering.
+// period.
+func (w *Watcher) Buffering(n Notifier) bool {
+	// add '&& w.Complete(n)' below to enable buffering only after the
+	// notifier/template has been completely evaluated/rendered.
```

Comment:
I added this comment as a reminder in case we needed a different behavior with the buffer... where it only buffers if it is considered "complete" (no dependencies w/o data) or if it buffers all the time. It currently does the latter and adding that '&& w.Complete(n)' makes it do the former.

Maybe I should tweak that comment to make the intention more clear?

Reply:
Ah, gotcha! I took the "add" to mean this was an imperative sentence. I think this interpretation is particular to me so no need to tweak anything. Thank you